autoload/airline/extensions/branch.vim

```diff
@@ -81,46 +81,39 @@ else
   endfunction
 endif
 
-let s:git_dirs = {}
 
-function! s:update_git_branch(path)
+" Fugitive special revisions. call '0' "staging" ?
+let s:names = {'0': 'index', '1': 'ancestor', '2':'target', '3':'merged'}
```

Comment:
After using this for a couple of days, I think the names used in merging are confusing. But then again, I virtually never merge, I always rebase, so if someone could look at if that makes sense. `'0':'index'` is fine, and I've gotten the other values from `:help fugitive-revision`